tools/pylint-extensions/azure-pylint-guidelines-checker/pylint_guidelines_checker.py

```diff
@@ -733,12 +895,16 @@ def visit_classdef(self, node):
         """
         try:
             if node.name.endswith("Client") and node.name not in self.ignore_clients:
-                if node.doc.find("code-block") != -1:
+                if node.doc_node.value.find("code-block") != -1:
```

Comment:
is this a breaking change with pylint 3?

Reply:
yes! It was deprecated adn removed